actions/types.go

```diff
@@ -0,0 +1,144 @@
+package actions
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+)
+
+type DatabaseCreateRequest struct {
+	Cluster  *CreateDBClusterInput
+	Instance *CreateDBInstanceInput
+}
+
+func (dcr DatabaseCreateRequest) String() string {
+	s, _ := json.MarshalIndent(dcr, "", "\t")
+	return string(s)
+}
+
+// CreateDBInstanceInput is the input for creating a new database instance
+// based on https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#CreateDBInstanceInput
+type CreateDBInstanceInput struct {
+	AllocatedStorage            *int64
+	AutoMinorVersionUpgrade     *bool
+	BackupRetentionPeriod       *int64
+	CopyTagsToSnapshot          *bool
+	DBClusterIdentifier         *string
+	DBInstanceClass             *string
+	DBInstanceIdentifier        *string
+	DBParameterGroupName        *string
+	DBSnapshotIdentifier        *string
+	DBSubnetGroupName           *string
+	EnableCloudwatchLogsExports []*string
+	Engine                      *string
+	EngineVersion               *string
+	MasterUserPassword          *string
+	MasterUsername              *string
+	MultiAZ                     *bool
+	Port                        *int64
+	PubliclyAccessible          *bool
+	StorageEncrypted            *bool
+	Tags                        []*Tag
+	VpcSecurityGroupIds         []*string
+}
```

Comment:
Do all of these things actually come from the user? offering?  I see a few I wonder about -- 
* AutoMinorVersionUpgrade
* EnableCloudwatchLogsExports
* PubliclyAccessible
* StorageEncrypted

Reply:
These come from the UI DB Controller

Reply:
Replaced some of those with opinionated defaults

Reply:
🎉